src/block.h

```diff
@@ -47,6 +47,7 @@ struct block {
     block_type type = block_untouched;
 
     surface_type surfs[face_count]{};
+    bool has_wire[face_count]{};
```

Comment:
How well does this extend to not just one wire? What are our options: we either break the world up into smaller chunks for wiring or we mark how many(?) wires are on a surface.